client/src/pages/Availability/AvailInterface.tsx

```diff
@@ -0,0 +1,154 @@
+import { Typography } from '@mui/material';
+import Grid from '@mui/material/Grid';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import { Field } from 'formik';
+import { classicNameResolver } from 'typescript';
+
+const Time = () => {
+  const output = [];
+  for (let i = 0; i < 24; i++) {
+    const listItem = <MenuItem value={`${i}:00`}>{`${i}:00`}</MenuItem>;
+    output.push(listItem);
+  }
+  return output;
+};
+
+const createAvailbilityInterface = (day: string, values: any, setFieldValue: any) => (
```

Comment:
```suggestion
const createAvailbilityRow = (day: string, values: any, setFieldValue: any) => (
```
Let's rename it to avoid any confusion as we're working with TS

Reply:
Done name change